test/parallel/test-cluster-kill-disconnect.js

```diff
@@ -1,5 +1,10 @@
 'use strict';
 const common = require('../common');
+
+// Check that cluster works perfectly for both `kill` and `disconnect` cases.
+// Also take into account that the `disconnect` event may be received after the
+// `exit` event.
```

Comment:
Nit: Maybe add a link to the GH issue that this test was named for originally?

Reply:
I don't know how I could have missed this, will do.

Reply:
Now I remember faintly. It was because I don't think if there was an issue in the first place.

All I could find corresponding to "3238" is https://github.com/nodejs/node-v0.x-archive/pull/3238. The PR actively talks about the test was edited in the PR, so I was wondering: should I add a link to the PR instead?